### PR TITLE
Last post :(

### PR DESCRIPTION
**Pull request by claucece**

### CODE REVIEW

File: blog/_posts/2015-10-15.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+title: Goodbye from team RubyGirlsQuito
+layout: post
+created_at: Tue Oct 13 2015.
```
Comment (alicetragedy) [inline blog/_posts/2015-10-15.md]: please change the date here to Thu Oct 15 2015 (without `.`)

File: blog/_posts/2015-10-15.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+permalink: blog/2015-10-15-goodbye-from-team-rubygirlsquito
+current: blog
+author: Sofi Cherenkov and Evy
+twitter: zeroknowledge9
+---
+
+So long and (and no thanks for the
+[externalities](http://dl.acm.org/citation.cfm?id=1719050) goodbye! Rails Girls Summer of Code
```
Comment (alicetragedy) [inline blog/_posts/2015-10-15.md]: there is a `)` missing here :)

**Comment by alicetragedy** [comment]
I've made a few notes above @claucece – could you make those changes and push them here?
I'd also ask you to rename your files as following: 
- the post so it has the date _and also_ your blog post title (so something like `2015-10-15-goodbye-from-team-rubygirlsquito.md`), as it helps us to track posts down more easily :)
- the images, something like `rubygirlsquito-goodbye-imagename.jpg` so we know which post they belong to. 
  Note: Of course, after changing the image names, you will have to update the links in the blog post so they still work ;)

After those changes are made I can happily merge the post. Thanks! :sparkles:

**Comment by claucece** [comment]
Hi, Laura! Everything has been changed! :tada:

**Pull request merged**